plat/arm/css/sgi/fdts/sgi575.dts

```diff
@@ -0,0 +1,11 @@
+/*
```

Comment:
Need confirmation that this doesn't break anything.

Reply:
Nariman, hope the required changes in the build scripts to package the dtb files have been made for the landing team build. If not, could you try doing that and check if you still see this issue?